### PR TITLE
Add Goerli to `evms` tables

### PR DESCRIPTION
**Pull request by hildobby**
# Thank you for contributing to Spellbook!
Please refer to the top of the `readme` in the root of Spellbook to learn how to contribute to Spellbook on DuneSQL.

### CODE REVIEW

**Comment by hildobby** [comment]
@MSilb7 u up? new chain about to drop in `evms`

**Comment by jeff-dude** [comment]
> @MSilb7 u up? new chain about to drop in `evms`

something i mentioned to @MSilb7 in dm's:
while we approved (and will continue to allow) the `evms` spells, the intention was to leave as a view and only use on the app/dashboards. we shouldn't use them as a shortcut source in spells downstream of these, i.e. for cross-chain spells. we should still follow the normal spell per blockchain (bonus if macro is used to centralize), then build cross-chain on top. it's how we can keep the processing/load time reasonable within spellbook.

i see that you aren't doing that in this PR, but just an fyi. we'll have to have a larger conversation if that is wanted in the future

File: models/evms/evms_transfers.sql [hunk 1]
```diff
@@ -0,0 +1,49 @@
+{{ config(
+        tags = ['dunesql'],
+        alias = alias('transfers'),
```
Comment (jeff-dude) [inline models/evms/evms_transfers.sql:3]: just double checking here --
you have alias `transfers` but the source tables below are `blocks`? is that correct?

**Comment by MSilb7** [comment]
> > @MSilb7 u up? new chain about to drop in `evms`
> 
> something i mentioned to @MSilb7 in dm's: while we approved (and will continue to allow) the `evms` spells, the intention was to leave as a view and only use on the app/dashboards. we shouldn't use them as a shortcut source in spells downstream of these, i.e. for cross-chain spells. we should still follow the normal spell per blockchain (bonus if macro is used to centralize), then build cross-chain on top. it's how we can keep the processing/load time reasonable within spellbook.
> 
> i see that you aren't doing that in this PR, but just an fyi. we'll have to have a larger conversation if that is wanted in the future

Yeah I'm a hard yes vote for `evms` to be a native table. There's too much overhead thinking of things as distinct chains now. I barely ever do "one chain only" queries anymore.

Especially as more L2s launch, segmenting by chain will make things completely unusable (imo).

**Comment by MSilb7** [comment]
> @MSilb7 u up? new chain about to drop in `evms`

I do wonder if we should add a 'chain_type' field natively in emvs (i.e. mainnet vs testnet) versus having someone need to join to evms_info

**Comment by jeff-dude** [comment]
> Yeah I'm a hard yes vote for `evms` to be a native table. There's too much overhead thinking of things as distinct chains now. I barely ever do "one chain only" queries anymore.
> 
> Especially as more L2s launch, segmenting by chain will make things completely unusable (imo).

raising internally, will see what we can do

**Comment by hildobby** [comment]
> > @MSilb7 u up? new chain about to drop in `evms`
> 
> I do wonder if we should add a 'chain_type' field natively in emvs (i.e. mainnet vs testnet) versus having someone need to join to evms_info

Yeah I thought about it, maybe a column like a boolean `is_mainnet` would be the most efficient usage-wise & storage-wise here?

File: models/evms/evms_transfers.sql (hunk 1 above)
Comment (hildobby) [reply models/evms/evms_transfers.sql]: that file isn't supposed to be here, just deleted it

**Review by jeff-dude — APPROVED** [review]
happy to proceed here, then you guys can change columns later if needed

**Pull request merged**